### DISCUSSION TITLE
Facial correction

### DISCUSSION THREAD

**Discussion by tangleps112** [lllyasviel/Fooocus#917] [Q&A]
When I am drawing large scenes, I often encounter facial distortions of characters. Is there any method for facial correction? For example, the AD plugin in WebUI, thank you.

**Comment by YacratesWyh** [comment]
From the scheduling problem of diffusion pointed out by google brain chen ting https://arxiv.org/abs/2301.10972, you should not generate small objects on large scenes, as a small noise means nothing for a big picture, thus only changes texture instead of linguistic canny. The better choice is to generate a normal portrait and generate outpaint scene, Or generate a large scene and clip the middle and inpaint the middle person, then paste back.

**Comment by adam-ah** [comment]
@YacratesWyh genius!
